forge-gui/res/cardsfolder/upcoming/the_balrog_durins_bane.txt

```diff
@@ -0,0 +1,12 @@
+Name:The Balrog, Durin's Bane
+ManaCost:5 B R
+Types:Legendary Creature Avatar Demon
+PT:7/5
+S:Mode$ ReduceCost | ValidCard$ Card.Self | Type$ Spell | Amount$ X | Relative$ True | EffectZone$ All | Description$ This spell costs {1} less to cast for each permanent sacrificed this turn.
+SVar:X:Count$SacrificedThisTurn Permanent
```

Comment:
outdated call

Reply:
Is this still wrong ? I've used Pyre-Sledge Arsonist as a template

Reply:
Yes syntax wrong slightly but logic would be wrong too because you ignore allies
Just use `PlayerCount$` 🤷‍♂️

Reply:
Also the `Relative$` doesn't make sense